app/models/manageiq/providers/amazon/cloud_manager/refresher.rb

```diff
@@ -12,6 +12,14 @@ def save_inventory(ems, target, _hashes_or_persister)
     EmsRefresh.queue_refresh(ems.ebs_storage_manager) if target.kind_of?(ManageIQ::Providers::BaseManager)
   end
 
+  def preprocess_targets
```

Comment:
this should not be needed? If we never queue it as a part of cloud manager?

Reply:
I'm not sure I understand what this bit is for.

Reply:
Yeah I don't think this is needed because by changing `combined_managers` this worker won't ever dequeue a refresh for the S3 manager

Reply:
@AlexanderZagaynov can we remove this method (it shouldn't be needed)

Reply:
@Ladas Have you seen that comment? https://github.com/ManageIQ/manageiq-providers-amazon/pull/461#issuecomment-409240480
Looks like it processes all the targets without current code.